roadmap.md

```diff
@@ -4,154 +4,95 @@ This document defines a high level roadmap for KubeEdge development.
 
 The [milestones defined in GitHub](https://github.com/kubeedge/kubeedge/milestones) represent the most up-to-date plans.
 
-The roadmap below outlines KubeEdge’s 2023 feature plan.
+The roadmap below outlines KubeEdge’s 2024 feature plan.
 
-## 2023 H1 
+## SIG Node
 
-### SIG Node
+- Continuous follow up Kubernetes release
+- Support edge nodes running on mac OS and RTOS 
+- Support in-cluster config for edge pods to access Kube-APIServer
+- Enhancements to device plugin on edge nodes, such as support for multiple virtual GPUs
+- Support event report to cloud 
+- Support for serverless computing
+- Remote maintenance
+- Add edge nodes in batches
```

Comment:
this is something enables user to add multiple edge nodes where edgecore runs all together? e.g we have edge servers running in the edge environment, and those servers can join cloudcore at the same time, so that we can avoid operating `join` process multiple times? i may be mistaken...

Reply:
> this is something enables user to add multiple edge nodes where edgecore runs all together? e.g we have edge servers running in the edge environment, and those servers can join cloudcore at the same time, so that we can avoid operating `join` process multiple times? i may be mistaken...

Yes, There is also a track issue: https://github.com/kubeedge/kubeedge/issues/5426